tokio/src/lib.rs

```diff
@@ -367,6 +367,12 @@
 //! rustflags = ["--cfg", "tokio_unstable"]
 //! ```
 //!
+//! <div class="warning">
+//! The <code>[build]</code> section does <strong>not</strong> go in a
+//! <code>Cargo.toml</code> file. Instead it must be placed in the Cargo config
+//! file <code>.cargo/config.toml</code>.
```

Comment:
Out of curiosity: can markdown be used here or does the wrapper div prohibit that?

Reply:
The wrapper div prevents it (I tried that first).